comp150-agile/essay.tex

```diff
@@ -0,0 +1,65 @@
+
+The aim of this being not only the resolutions of the barriers by the scrum master, but to keep the team informed on the general progress of the other members and the project as a whole. 
+
+According to various online guides  \cite{ DailyScrum, EffectiveScrum} the role of the problem solver is left to the scrum master. In contrast to this, professionals in other fields describe an effective meeting to draw "upon each members' knowledge, skills, and perspectives to solve problems and to support one another" \cite{LeadershipLesson}. Research in relevant fields found that "Relationships in virtual teams are developed and strengthened through a proactive effort to solve problems" \cite[p. 559]{tavvcar2005skills}. 
+
+Researchers in Psychology have shown that teams perform better the greater team cooperation and helping behaviour. Teams composed of more altruistic individuals, do better than those composed of more selfish ones \cite[p. 542]{li2014toward}.  The same researcher feels that some "work team setting also provides opportunities for team members to behave egoistically and to promote self-interest"\cite[p. 542]{li2014toward}. The author feels that properly conducted scrum meetings would reduce the negative impact of individual team members that lean towards self-interest. Other authors have noted that just one disagreeable member of the team can have a negative impact on the rest of the team "thereby destroy interpersonal relationships" \cite[p. 381] {barrick1998relating}.
+
+Within the games industry, the range of skills needed to create any game is wildly diverse. From artists to game designers to engine programmers. With such a range, conflict will occur at some point\cite{gorse2007communication}. One type of conflict is Relationship conflict, this conflict "is interpersonally focused" \cite [p. 215]{costa2015direct} and research describes it to stem from "disagreements about personality differences, different values, or different norms" \cite [p. 215]{costa2015direct}. While this may mean relations within the team are suffering, the same author also argues that conflict may benefit a team as it could allow "teams to consider a broader range of solutions" and "reduces the likelihood of thought processing biases"\cite [p. 215]{costa2015direct}. The strict framework of a scrum meeting reduces the likely hood of such conflict arising as it reduces the influence of personalities to the reports. In contrast your more traditional meeting has much more flexibility, allowing the different values and personalities of the team to influence. While the strict framework of scrum meetings may reduce conflict, the author believes it also limits the interpersonal growth. This is due to the fact that the barriers are left for the scrum master to solve, meaning the team misses out on a potential bonding exercise and use of team expertise to fix potential problems.

```

Comment:
I'm not sure that the team misses out on bonding exercise, since a good scrum master should be learning everyone's skills and preferences and be able to pair/mob people together to facilitate such bonding. Also, most people in the industry are adults and will bond of their own volition.